src/utils/getUsersById.ts

```diff
@@ -0,0 +1,4 @@
+import usersFromServer from '../api/users';
+
+export const getUsersById = (id: number) =>
+  usersFromServer.find(user => user.id === id);
```

Comment:
The `getUsersById` function might return `undefined` if no user is found with the given `id`. This could cause issues when rendering components that expect a valid user object. Consider adding a check or providing a default value to handle cases where the user is not found.